server/plugin.go

```diff
@@ -81,13 +85,31 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 	p.API.RegisterCommand(getCommand())
-	user, err := p.API.GetUserByUsername(config.Username)
+
+	botId, appErr := p.Helpers.EnsureBot(&model.Bot{
```

Comment:
Late catch: This is potential dangerous. `EnsureBot` returns an `error`, not `*model.AppErr`. Because of this the checks later can fail. @crspeller Would you please fix this?